sidebars.ts

```diff
@@ -1090,7 +1090,6 @@ module.exports = {
             'alerts/monitors/muting-schedules',
             'manage/ingestion-volume/monitor-ingestion-receive-alerts',
             'alerts/monitors/use-playbooks-with-monitors',
-            'alerts/monitors/automation-payload-variables'
```

Comment:
should docs/platform-services/automation-service/automation-service-playbooks.md be added to` platform-services `sidebars?

Reply:
@kimsauce, that file is already in the platform services sidebar:
https://help.sumologic.com/docs/platform-services/automation-service/automation-service-playbooks/